backend/src/server/routes/v2/index.ts

```diff
@@ -21,6 +22,7 @@ export const registerV2Routes = async (server: FastifyZodProvider) => {
     async (projectServer) => {
       await projectServer.register(registerProjectRouter);
       await projectServer.register(registerIdentityProjectRouter);
+      await projectServer.register(registerProjectMembershipRouter);
```

Comment:
Last I spoke to Tony, I was told that we'd be doing a "larger" move at a later time, where we'll properly start renaming things to the `project` namespace. The reason for going with the `v2` router, is because the `v1` router would have conflicting endpoints if we use the `workspace` namespace. I think if we're planning to do a larger renaming later, we should keep this on the `v2` router for this.